phpBB/phpbb/feed/controller/feed.php

```diff
@@ -386,7 +386,7 @@ protected function send_feed_do(feed_interface $feed)
 		));
 
 		$response = new Response($content);
-		$response->headers->set('Content-Type', 'application/atom+xml');
+		$response->headers->set('Content-Type', 'application/atom+xml; charset=UTF-8');
 		$response->setCharset('UTF-8');
```

Comment:
Isn't this line doing this already?

Reply:
And for the reference about reasons it works the way like that: https://github.com/symfony/symfony/issues/13410

Reply:
I left it untouched in order to prevent some hypothetical situation when we will rely on charset property somewhere inside response class (well, m.b. sometimes in future?)
But I'm not sure if I'm right, because it sounds like overkill - in  https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/HttpFoundation/Response.php#L283  Symphony set UTF-8 charset if it isn't defined yet
Maybe it is really worth to remove setCharset here - it's up to you)

Reply:
Gentle ping)

Reply:
Symfony defaults to UTF-8, so the setCharset() call here should be removed.

Reply:
> Symfony defaults to UTF-8, so the setCharset() call here should be removed.

Ok, I removed it)